webpack/components/Table/EmptyStateMessage.js

```diff
@@ -97,7 +97,8 @@ const EmptyStateMessage = ({
           </EmptyStateSecondaryActions>
         }
 
-        {(showSecondaryActionButton || searchIsActive || !!filtersAreActive) &&
+        {(!showSecondaryActionAnchor &&
```

Comment:
Added this so that it only shows one secondary action - either the <a> link style or the button with onClick - but not both.